specification/ai/Azure.AI.Projects/datasets/models.tsp

```diff
@@ -37,9 +37,11 @@ model DatasetVersion {
   @doc("Dataset type")
   type: DatasetType;
 
-  @doc("Indicates if dataset is reference only or managed by dataset service. If true, the underlying data will be deleted when the dataset version is deleted")
-  @visibility(Lifecycle.Read)
-  isReference?: boolean;
+  @doc("Indicates if the dataset holds a reference to the storage, or the dataset manages storage itself. If true, the underlying data will not be deleted when the dataset version is deleted")
+  isReference: boolean;
+
+  @doc("The Azure Storage Account connection name. Required if isReference is true.")
+  connectionName?: string;
```

Comment:
```suggestion
  @doc("Indicates if the dataset holds a reference to the storage, or the dataset manages storage itself. If true, the underlying data will not be deleted when the dataset version is deleted")
  @visibility(Lifecycle.Read, Lifecycle.Create)
  isReference: boolean;

  @doc("The Azure Storage Account connection name. Required if isReference is true.")
  @visibility(Lifecycle.Read, Lifecycle.Create)
  connectionName?: string;
```

Reply:
So that in the update payload, these won't show up (datasets are immutable once created)